src/RadioButton/index.js

```diff
@@ -0,0 +1,121 @@
+// @flow
+
+import * as React from "react";
+import { darken } from "polished";
+import * as tokens from "orbit-design-token";
+
+type Props = {
+  name: string,
+  label: string,
+  value: string,
+  error?: boolean,
```

Comment:
`hasError` would sound better IMHO, without looking on types, I would expect `error` is some string or Error object

Reply:
Good point, I'll use `hasError`.